### DISCUSSION TITLE
Error Format

### DISCUSSION THREAD

**Discussion by mahyarkarimi** [keystonejs/keystone#5726] [Questions]
Are the errors thrown from apollo server exactly like the documentations in https://www.apollographql.com/docs/apollo-server/data/errors .
I think of there was an option to make production server errors data to be masked from containing stacktrace.

**Comment by bladey** [comment]
Hi @mahyarkarimi, error reporting has recently been improved, including the ability to see Stacktraces, which is configurable. Please join us on https://community.keystonejs.com if you have any further questions!
